tools/infrasys/src/root.rs

```diff
@@ -0,0 +1,195 @@
+use super::{error, KeyRole, Result};
+use log::{trace, warn};
+use pubsys_config::SigningKeyConfig;
+use rusoto_core::Region;
+use snafu::{ensure, OptionExt, ResultExt};
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+
+/// The tuftool macro wraps Command to simplify calls to tuftool, adding region functionality.
+macro_rules! tuftool {
+    ($region:expr, $format_str:expr, $($format_arg:expr),*) => {
+        let arg_str = format!($format_str, $($format_arg),*);
+        trace!("tuftool arg string: {}", arg_str);
+        let args = shell_words::split(&arg_str).context(error::CommandSplit { command: &arg_str })?;
+        trace!("tuftool split args: {:#?}", args);
+
+        let status = Command::new("tuftool")
+            .args(args)
+            .env("AWS_REGION", $region)
+            .status()
+            .context(error::TuftoolSpawn)?;
+
+        ensure!(status.success(), error::TuftoolResult {
+            command: arg_str,
+            code: status.code().map(|i| i.to_string()).unwrap_or_else(|| "<unknown>".to_string())
+        });
+    }
+}
```

Comment:
We should try to dedup this if possible. Can the existing macro be used if it is moved to pubsys-config and exported as part of that lib?

i.e. move this and use it?
https://github.com/bottlerocket-os/bottlerocket/blob/develop/tools/pubsys-setup/src/main.rs#L51...L71

Reply:
Turns out that there is a subtle difference in these two macros that makes de-duplicating arguably more trouble than it's worth. Resolving.